repository-structure.md

```diff
@@ -0,0 +1,75 @@
+# Common Repository Structure
+
+Hyperledger projects are required to maintain a standard set of files in each repository. This
+document describes the required and recommended files.
+
+## Required Files with Specified Content
+
+Repositories MUST have these files with the specific content in the linked files, or a file with a
+link to the specified content with minimal exposition. These files MUST be at the root of the
+repository.
+
+-   [`LICENSE`](https://www.apache.org/licenses/LICENSE-2.0.txt) - https://www.apache.org/licenses/LICENSE-2.0.txt\
```

Comment:
[Just noticed] Do we need to specify it as either `Apache 2.0` or `Creative Commons`?

Reply:
This is intended for code repos and createive commons is not relevant to code.